### PR TITLE
Fix #1764 - Removed 'main.xml' from the menu folder.

### LINKED ISSUE #1764: Remove unused file "main.xml"
**Actual Behaviour**

Unused file "main.xml" is present.

**Expected Behaviour**

File should not be present.

**Steps to reproduce it**

Remove the aforementioned file.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by shivam1997**
Fixed #1764 

Changes: 
Removed "main.xml" file from the "menu" folder.
Commented the one usage of the removed file in "CameraActivity.java" file.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java [hunk 1]
```diff
@@ -609,12 +609,12 @@ protected void onDestroy() {
         super.onDestroy();
     }
 
-    @Override
+    /*@Override
     public boolean onCreateOptionsMenu(Menu menu) {
         // Inflate the menu; this adds items to the action bar if it is present.
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java]: Remove the unused code completely instead of commenting.

**Comment by shivam1997** [comment]
@anantprsd5 made the changes as requested by you, please review it again.

**Comment by shivam1997** [comment]
@anantprsd5 please review it.

**Comment by shivam1997** [comment]
@anantprsd5, @isuruAb please review it.

**Comment by shivam1997** [comment]
@anantprsd5, @isuruAb please review & merge it.

**Comment by abishekvashok** [comment]
Can you check the failing Travis ci build?

**Comment by abishekvashok** [comment]
@shivam1997 i am going to close this and keep it on hold. When you are free to work on it just ping here. :)

**Comment by shivam1997** [comment]
Reopen this. I would to work on it.

**Pull request closed without merging**